hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -198,10 +203,7 @@ private HoodieWriteConfig createMetadataWriteConfig(HoodieWriteConfig writeConfi
         default:
           throw new HoodieMetadataException("Unsupported Metrics Reporter type " + writeConfig.getMetricsReporterType());
       }
-
-      builder.withMetricsConfig(metricsConfig.build());
     }
-
```

Comment:
keep this empty line?

Reply:
The extra blank lines should be deleted here